unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/RPCSocialApiBridgeShould.cs

```diff
@@ -44,20 +44,21 @@ public IEnumerator SetUp()
 
             var friendshipsService = Substitute.For<IFriendshipsService<MockSocialServerContext>>();
 
-            friendshipsService.GetFriends(Arg.Any<Payload>(), Arg.Any<MockSocialServerContext>())
-                              .Returns(callInfo =>
-                               {
-                                   MockSocialServerContext context = (MockSocialServerContext)callInfo[1];
-
-                                   return UniTaskAsyncEnumerable.Create<Users>(async (writer, token) =>
-                                   {
-                                       foreach (var users in context.userList)
-                                       {
-                                           if (token.IsCancellationRequested) break;
-                                           await writer.YieldAsync(users);
-                                       }
-                                   });
-                               });
+            // TODO: fix this error CS1660: Cannot convert lambda expression to type 'IUniTaskAsyncEnumerable<UsersResponse>' because it is not a delegate type
```

Comment:
https://github.com/decentraland/protocol/commit/9a568b16b2eafb134177329ba670c1451be8a169 This was merged but it didn't have its coding here.